### PR TITLE
Push nightly AWS Codegen changes direct to Master

### PR DESCRIPTION
**Pull request by onno-vos-dev**
Same change as done in: https://github.com/aws-beam/aws-erlang/pull/100

### CODE REVIEW

**Review by philss — APPROVED** [review]
Looks good! :shipit: 

I wish we could keep the reference for the original release in the commit message, but I think it's fine this way :+1:

**Comment by onno-vos-dev** [comment]
> Looks good! :shipit:
> 
> I wish we could keep the reference for the original release in the commit message, but I think it's fine this way 👍

Which reference do you mean? :) Maybe we can sort that out?

**Comment by philss** [comment]
> Which reference do you mean? :) Maybe we can sort that out?

It was only in the PR message. Like we have in the last one: https://github.com/aws-beam/aws-elixir/pull/137

> This is an automated pull request.
It was triggered GitHub actions. The details can be found at
https://github.com/aws-beam/aws-elixir/actions/workflows/codegen.yml
>
> For changes, please check https://github.com/aws/aws-sdk-go/releases/tag/v1.43.16

I will left a suggestion, 1 sec.

File: .github/workflows/codegen.yml [hunk 1]
```diff
@@ -99,26 +99,16 @@ jobs:
...
 
-      - name: Open Pull Request
-        uses: peter-evans/create-pull-request@v3
+      - name: Commit files
+        run: |
+          git config --local user.email "noreply@github.com"
+          git config --local user.name "github-actions[bot]"
+          git add lib/aws/generated/*
+          git add .latest-tag-aws-sdk-go
+          git commit -m "Update services based on ${{ env.LATEST_AWS_SDK_GO_TAG }} of AWS Go SDK"
```
Comment (philss) [inline .github/workflows/codegen.yml:108]:
```suggestion
          echo "Update services based on ${{ env.LATEST_AWS_SDK_GO_TAG }} of AWS Go SDK" >> commit-msg
          echo >> commit-msg
          echo "Reference: https://github.com/aws/aws-sdk-go/releases/tag/${{ env.LATEST_AWS_SDK_GO_TAG }}" >> commit-msg
          git commit -F commit-msg
```

This is slightly more complicated but does the trick.

Comment (onno-vos-dev) [reply .github/workflows/codegen.yml]: Looks good! I'll apply the same change to aws-erlang later 👍 Thanks for the fast feedback 🙏

**Pull request merged**